ci/docker/runtime_functions.sh

```diff
@@ -739,6 +739,9 @@ unittest_ubuntu_gpu_cpp() {
 unittest_ubuntu_cpu_R() {
     set -ex
     mkdir -p /tmp/r-site-library
+    # build R packages in parallel
+    mkdir -p ~/.R/
+    echo  "MAKEFLAGS = -j72" > ~/.R/Makevars
```

Comment:
Since this is during runtime, we can us nproc instead of hardcoding the number of jobs